pages/docs/visualizing-data/controls/pivot-table-controls.mdx

```diff
@@ -2,8 +2,18 @@
 description: An overview of what is available in the control panel for configuring Pivot Table visualizations.
 ---
 
+import { Callout } from "nextra-theme-docs";
+
 # Pivot Table controls
 
+<Callout type="warning">
+  Support for pivot tables will be ending in the near future, as tables can now
+  specify columns to pivot on. <br /> <br/>
+  Pivot tables should only be used for pivoting on a broken out attribute, which is currently unsupported in tables. 
```

Comment:
This sentence is basically saying the same thing as the next sentence, but less clear. Just remove it I think.